v3/integrations/logcontext-v2/nrslog/attributes.go

```diff
@@ -0,0 +1,97 @@
+package nrslog
+
+import (
+	"log/slog"
+	"maps"
+	"strings"
+)
+
+type attributeCache struct {
+	preCompiledAttributes map[string]interface{}
```

Comment:
Recommend using `any` instead of `interface{}` these days as a style point, but I wouldn't hold up the review for that.